CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # moi changelog
 
-## Version 0.1.0-dev, changes go here
+## Version 0.1.0-dev
+
+Currently in development, released on May 2nd 1023
```

Comment:
Perhaps the most important event that year besides [The Ghaznavid Empire's occupation of Transoxiana](http://en.wikipedia.org/wiki/1023#Asia)